### PR TITLE
SW-4730 Add 'not present' filter to featured filters used in new project creation flow and inventory tables. Implement 'not present' filter for the projects featured filter

### PR DESCRIPTION
**Pull request by nickgraz**
Inventory View

Project Create flow

### CODE REVIEW

**Comment by nickgraz** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #2102** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2102?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2102?utm_source=stack-comment).

File: src/components/ProjectNewView/flow/ProjectEntityFilter.tsx [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+import React, { useCallback, useMemo, useState } from 'react';
 import { Popover, Theme, Typography } from '@mui/material';
 import { makeStyles } from '@mui/styles';
 import Icon from 'src/components/common/icon/Icon';
 import useDeviceInfo from 'src/utils/useDeviceInfo';
 import FilterMultiSelect from 'src/components/common/FilterMultiSelect';
 import { ProjectEntityFilters } from 'src/components/ProjectNewView/flow/useProjectEntitySelection';
 import { PillListItemWithEmptyValue } from 'src/components/ProjectNewView/flow/ProjectEntitySearch';
+import strings from '../../../strings';
+import { useLocalization } from '../../../providers';
```
Comment (karthikbtf) [inline src/components/ProjectNewView/flow/ProjectEntityFilter.tsx:10]: src/strings, src/providers

File: src/components/common/FilterMultiSelect/index.tsx [hunk 2]
```diff
@@ -103,7 +119,28 @@ export default function FilterMultiSelect<T>(props: FilterMultiSelectProps<T>):
...
+              name={`not-preset-${String(filterKey)}`}
+              label={notPresentFilterLabel}
+              onChange={() => {
+                const nextIsNotPresentFilterSelected = !isNotPresentFilterSelected;
+                if (nextIsNotPresentFilterSelected) {
+                  setSelection([null]);
+                } else {
+                  setSelection([]);
+                }
+                setIsNotPresentFilterSelected(!isNotPresentFilterSelected);
```
Comment (karthikbtf) [inline src/components/common/FilterMultiSelect/index.tsx:138]: can use `nextIsNotPresentFilterSelected`

**Review by karthikbtf — APPROVED** [review]
Cool!!

**Comment by nickgraz** [comment]
### Merge activity

* **Feb 2, 10:46 AM EST**: @nickgraz started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2102).

**Pull request merged**